databricks/koalas/series.py

```diff
@@ -452,6 +452,10 @@ def spark_type(self):
     def add(self, other) -> "Series":
         return self + other
 
+    def __add__(self, other):
+        other = ks.Index(other, name=self.name) if isinstance(other, (list, tuple)) else other
```

Comment:
- Why do we create Index to compare instead of Series?
- Looks like Index support this too. Can't we move this into `base.py`?

Reply:
> * Why do we create Index to compare instead of Series?

If we use Series here, we cannot match the Index as below.

```python
>>> kser = ks.Series([1, 2, 3, 4], index=[10, 20, 30, 40])
>>> kser + ks.Series([3, 4, 5, 6])
0    NaN
1    NaN
10   NaN
3    NaN
2    NaN
30   NaN
20   NaN
40   NaN
dtype: float64
```

so we use Index.

```python
>>> kser + ks.Index([3, 4, 5, 6])
10     4
20     6
30     8
40    10
dtype: int64
```

> * Looks like Index support this too. Can't we move this into `base.py`?

Yes, think we can. Let me move this into `base.py`

Thanks!